src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/OpenMethodResolver.cs

```diff
@@ -61,6 +65,7 @@ public OpenMethodResolver(RuntimeTypeHandle declaringType, IntPtr codePointer, G
             _declaringType = declaringType.ToEETypePtr();
             _handle = handle;
             _readerGCHandle = readerGCHandle;
+            s_lazyGvmLookupForSlot = &TypeLoaderExports.GVMLookupForSlot;
```

Comment:
Do we really need to set it here? The type loader call is under `if (_resolveType == GVMResolve)`, but we are setting the resolver to `OpenNonVirtualResolve` in this method. I would expect this pointer to be only set when we set `_resolveType = GVMResolve`.

Reply:
We don't - my concern was that if someone adds a new ctor, how do we make it less likely they'll forget to initialize this static? Person adding it _might_ look at one of the other ctors. But I don't have a strong opinion on this.

Reply:
This risk can be mitigated by a comment (suggested one).